src/net.cpp

```diff
@@ -2211,7 +2211,7 @@ bool CConnman::Start(CScheduler& scheduler, std::string& strNodeError, Options c
 
     clientInterface = connOptions.uiInterface;
     if (clientInterface)
-        clientInterface->InitMessage(_("Loading addresses..."));
+        clientInterface->InitMessage(_("Loading p2p addresses from peers.dat..."));
```

Comment:
No need to mention the file name in a translated message. Also missing curly braces {} according to style guide.